modules/components/pages/outputs/azure_data_lake_gen2.adoc

```diff
@@ -0,0 +1,123 @@
+= azure_data_lake_gen2
+//tag::single-source[]
+:type: output
+:page-beta: true
+
+:categories: ["Services","Azure"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
```

Comment:
In some cases, I've updated the wording of the field descriptions. Please check carefully.

Reply:
All of the fields should be supported in cloud.